### PR TITLE
magento/devdocs#: Cloud. Add notice about “wizard:ideal-state”

### PR DESCRIPTION
**Pull request by atwixfirster**
## Purpose of this pull request

This pull request (PR) adds explanation why `./vendor/bin/ece-tools wizard:ideal-state` always returns `The configured state is not ideal` error on local instance.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.2/cloud/reference/ece-tools-reference.html
- https://devdocs.magento.com/guides/v2.3/cloud/reference/ece-tools-reference.html
- https://devdocs.magento.com/guides/v2.2/cloud/deploy/smart-wizards.html
- https://devdocs.magento.com/guides/v2.3/cloud/deploy/smart-wizards.html

## Links to Magento source code

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
Hi @atwixfirster, thank you for your collaboration. It would be great to shed more technical light on why exactly this command behaves in this way locally. But this simple hint is also useful. In case if you decide to dig this issue deeper, you have a place to share your thoughts.

File: guides/v2.2/cloud/deploy/smart-wizards.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Each of the smart wizard commands provides a verification response and, if appli
 
 Command | Description
 ------- | -----------
-`wizard:ideal-state` | Check that SCD is on the _build_ stage, the `SKIP_HTML_MINIFICATION` variable is `true`, and the post_deploy hook configured.
+`wizard:ideal-state` | Check that SCD is on the _build_ stage, the `SKIP_HTML_MINIFICATION` variable is `true`, and the post_deploy hook configured. Do not run locally.
```
Comment (hguthrie) [inline guides/v2.2/cloud/deploy/smart-wizards.md]:
```suggestion
`wizard:ideal-state` | Check that SCD is on the _build_ stage, the `SKIP_HTML_MINIFICATION` variable is `true`, and the post_deploy hook configured in the Cloud environment. Not for use in the local development environment.
```

File: guides/v2.2/cloud/reference/ece-tools-reference.md [hunk 2]
```diff
@@ -95,6 +95,9 @@ There is a set of verification commands available to help evaluate the configura
 php ./vendor/bin/ece-tools wizard:ideal-state
 ```
 
+{: .bs-callout-info }
+Run `wizard:ideal-state` on the Cloud environment only. Command execution on the local environment always returns `The configured state is not ideal` error.
```
Comment (hguthrie) [inline guides/v2.2/cloud/reference/ece-tools-reference.md]:
```suggestion
You must run the `wizard:ideal-state` command in the Cloud environment. The command always returns the `The configured state is not ideal` error when run in the local development environment.
```

**Comment by hguthrie** [comment]
Needs a couple of adjustments. Nothing serious, just some language nits. Looks good, otherwise. :)

**Comment by dshevtsov** [comment]
@atwixfirster please sync your branch with `master` to resolve the conflicts.

File: guides/v2.2/cloud/deploy/smart-wizards.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.2/cloud/deploy/smart-wizards.md]: **implemented**

File: guides/v2.2/cloud/reference/ece-tools-reference.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.2/cloud/reference/ece-tools-reference.md]: **implemented**

Thank you, @hguthrie

**Comment by atwixfirster** [comment]
> @atwixfirster please sync your branch with `master` to resolve the conflicts.

**done**

**Comment by hguthrie** [comment]
running tests

**Comment by hguthrie** [comment]
running tests

**Pull request merged**